src/webgl/p5.Geometry.js

```diff
@@ -147,6 +156,69 @@ p5.Geometry.prototype.averagePoleNormals = function() {
 };
 
 /**
+ * Create a 2D array for establishing stroke connections
+ * @return {p5.Geometry}
+ */
+p5.Geometry.prototype._makeTriangleEdges = function() {
+  if (Array.isArray(this.strokeIndices)) {
+    for (var i=0, max=this.strokeIndices.length; i<max; i++) {
+      this.edges.push(this.strokeIndices[i]);
+    }
+  }
+  else {
+    for (var j=0, amt=this.faces.length; j<amt; j++) {
+      this._addEdge(this.faces[j][0], this.faces[j][1]);
+      this._addEdge(this.faces[j][1], this.faces[j][2]);
+      this._addEdge(this.faces[j][2], this.faces[j][0]);
+    }
+  }
+  return this;
+};
+
+p5.Geometry.prototype._addEdge = function(i, j) {
+  var edge = [i, j];
+  this.edges.push(edge);
+};
+
+/**
+ * Create 4 vertices for each stroke line, two at the beginning position
+ * and two at the end position. These vertices are displaced relative to
+ * that line's normal on the GPU
+ * @return {p5.Geometry}
+ */
+p5.Geometry.prototype._edgesToVertices = function() {
+  this.lineVertices = [];
+  var vertices = this.lineVertices;
```

Comment:
IIRC, we need to capture a reference to the vertices variable so it is properly scoped in the closure that pushes data to it (as the meaning of `this` changes).

Reply:
Yeah that is the reason for doing that before but I am changing it quite a bit right now and it looks more like `this.lineVertices.push(a, b, c, c, b, d);`. I convert them to arrays earlier in the code and that way I can get rid of the `store` function altogether.

Reply:
ah got it